<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemesParamWidget</class>
 <widget class="QWidget" name="DemesParamWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Selected demes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="activatedLabel">
        <property name="text">
         <string>Activated :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="carryingCapacityLabel">
        <property name="text">
         <string>Carrying capacity :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="growthRateLabel">
        <property name="text">
         <string>Growth rate :</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="groupLabel">
        <property name="text">
         <string>Group :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="activatedCheckBox">
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="ABCIntervalIntWidget" name="carryingCapacityABC" native="true"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="sampleSizeLabel">
        <property name="text">
         <string>Sample size :</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="groupComboBox"/>
      </item>
      <item row="9" column="1">
       <widget class="QSpinBox" name="sampleSizeSpinBox">
        <property name="specialValueText">
         <string>Varying</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="migrationRateLabel">
        <property name="text">
         <string>Migration rate :</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="ABCIntervalDoubleWidget" name="growthRateABC" native="true"/>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="ABCIntervalDoubleWidget" name="migrationRateABC" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="initialPopulationLabel">
        <property name="text">
         <string>Initial population :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="ABCIntervalIntWidget" name="initialPopulationABC" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ABCIntervalIntWidget</class>
   <extends>QWidget</extends>
   <header>gui/ABCIntervalIntWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ABCIntervalDoubleWidget</class>
   <extends>QWidget</extends>
   <header>gui/ABCIntervalDoubleWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
